src/components/backend-ai-general-styles.ts

```diff
@@ -129,11 +126,14 @@ export const BackendAiStyles = [
       --general-progress-bar-bg: var(--token-colorBorderSecondary, #e8e8e8);
       --general-progress-bar-reserved: linear-gradient(
         to left,
-        #722cd7,
-        #5c7cfa
+        var(--token-blue-8) 0%,
```

Comment:
@yomybaby 
This is the part that controls the gradient color of the progress bar.

Reply:
I pushed a commit to remove gradient color of progress bar.